srcpkgs/cowpatty/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cowpatty'
+pkgname=cowpatty
+version=4.6
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+disable_parallel_build=1
+make_use_env=1
+LDFLAGS="-I /usr/share/include/"
```

Comment:
This doesn't make any sense.

1. Includes should be in CFLAGS
2. I compiled just fine without it (natively and aarch64-musl)

Reply:
I changed it, Apparently it only works with make_use_env=1. Previously without that and the include it failed.